move object properties from shared to original modules #1703

stap-m opened the pull request:
## Summary of the discussion

comes from #1703 

## Type of change (CHANGELOG.md)

### Updated
- place to live for several domain and ranges of object properties in oeo-shared

### Removed
- doubles axioms in oeo-shared

## Workflow checklist

### PR-Assignee
- [ ] 🐙 Follow the [Pull Request Workflow](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow)
- [ ] 📝 Update the [CHANGELOG.md](https://github.com/OpenEnergyPlatform/ontology/blob/dev/CHANGELOG.md)
- [ ] 📙 Add #'s to `term tracker item`

### Reviewer
- [ ] 🐙 Follow the [Reviewer Guide](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow#reviewer-guide-check-changes-introduced-by-a-pull-request)
- [ ] 🐙 Provided feedback and show sufficient appreciation for the work done

stap-m commented:
`has client`: Domain (`model` or `information content entity`) lives in oeo-model, range (`'has role' some client`) lives in oeo-social. I see two options:
1. I move whole the object property to oeo-shared-axiom 
2. I move the object property to oeo-social and the domain declaration to oeo-shared-axioms

Any opinions?

stap-m commented:
`has bearer` and `has characteristic` live in oeo-shared and must not be moved. They have `energy` as domain and range resp. Again two options:
1. move energy to oeo-shared (In this case my preferred option)
2. split domain and range from class declaration and transfer it to oeo-physical

stap-m commented:
I leave `covers sector` untouched in oeo-shared. We can take care of it in #1703

l-emele commented:
Thinking about this, I am asking myself, why is `model` not a subclass of `information content entity` (ICE)? The definition of ICE is: _A generically dependent continuant that is about some thing._ To me, that definition holds also to model, as a model is about an _idealised reproduction of some system and its behaviours_.

If we make model a subclass of ICE, we can simplify the domain of `has contributor` to ICE and we don't have any problem in OEO shared.

stap-m commented:
There is a discussion in #1444 about the classification of model with the colclusion to make it equivalent to `continuant and 'has role' some 'model role'`

stap-m commented:
. [ ] We have a lost object property OEO_00010128 that needs to be restored.

stap-m commented:
That's it so far for shared. 
The other modules should be done in separate PRs.

stap-m commented:
> * [x]  `has bearer` and `has characteristic` live in oeo-shared and must not be moved. They have `energy` as domain and range resp. Again two options:
> 
> 
>     1. move energy to oeo-shared (In this case my preferred option)
> 
>     2. split domain and range from class declaration and transfer it to oeo-physical

I just implemented this now to complete the PR. If there are good reasons, we can still revert.

stap-m commented:
Note for reviewers: 
I merged #1709 and #1710 into this branch. The've been already reviewed.  
All commits until [c21f357](https://github.com/OpenEnergyPlatform/ontology/pull/1706/commits/c21f357237a4e5794f1878f4e64e527e867ac689) are not yet reviewed.

stap-m commented:
@l-emele @areleu now the PR should be fine. Please take a hopefully final look.

areleu commented:
At first glance there seems not to be any duplicated axioms, the object property tree looks clean.
